docs/install_ovos_core.md

```diff
@@ -1,70 +1,185 @@
+**NOTE** The GUI requires seperate packages in addition to what is required by ovos-core.  The [GUI installation](install_gui.md) is covered in its own section.
 
-All Mycroft Skills and Plugins should work normally with OVOS-core. 
+## Host system
+OVOS requires some system dependencies, how to do this will depend on your distro
 
-OVOS-core is fully modular. Furthermore, common components have been repackaged as plugins. That means it isn't just a great assistant on its own, but also a pretty small library!
+Ubuntu/Debian based images
 
+```
+sudo apt install build-essential python3-dev python3-pip swig libssl-dev libfann-dev portaudio19-dev libpulse-dev
+```
 
-## Getting Started
+A few packages are not necessasary, but are needed to install from source and may be required for some plugins.  To add these packages run this command.
 
-ovos-core is very modular, depending on where you are running ovos-core you may want to run only a subset of the services
+```
+sudo apt install git libpulse-dev cmake libncurses-dev pulseaudio-utils pulseaudio
+```
 
-by default ovos-core only installs the minimum components common to all services, for the purposes of this document we will assume you want a full install
+**NOTE**: MycroftAI's `dev_setup.sh` does not exist in OVOS-core.  See the community provided, **WIP**, [manual_user_install](https://github.com/OpenVoiceOS/raspbian-ovos/blob/dev/manual_user_install.sh) for a minimal, almost, replacement.
 
-if you want to finetune the components please replace `[all]` in commands below with the subset of desired extras, eg `[skills,bus]`
+## Get started
+We suggest you do this in a virtualenv.
 
+Create and activate the virtual environment
+```
+python -m venv .venv
+. .venv/bin/activate
+```
+Update `pip` and install `wheel`
 
-## Installing ovos-core
+`pip install -U pip wheel`
 
-ovos-core can be installed from pypi or from source
+## From PyPi
 
-if install fails you may need to install some system dependencies, how to do this will depend on your distro
+### ovos-core
+To install a full OVOS software stack with enough skills and plugins to have a working system, the OVOS team includes a subset of packages that can be installed automatically with pip
 
-```bash
-sudo apt install build-essential python3-dev swig libssl-dev libfann-dev portaudio19-dev libpulse-dev
-```
+**NOTE** This will install `lgpl` licenced software.
 
-**Note**: MycroftAI's `dev_setup.sh` does not exist in OVOS-core.
+It is recommended to use the latest alpha versions until the `0.1.0` release as it contains all of the latest bug fixes and improvements.
 
-### from source
+**alpha version**
 
-We suggest you do this in a virtualenv:
+`pip install --pre ovos-core[mycroft]`
 
-`pip install git+https://github.com/OpenVoiceOS/ovos-core[all]`
+**latest stable**
 
-### from pypi
+ovos-core 0.0.7 does not include the new `extras [mycroft]` so we use `[all]`
```

Comment:
Consider putting this first, since 0.0.7 has been out for a while and 0.0.8 should be released in the next couple of months.